etc/syncd.d/msn2700_port_map.ini

```diff
@@ -0,0 +1,33 @@
+# alias          lanes
```

Comment:
do we want to put the file here or use ansible to deploy the file like this and pass the path to the daemon